scripts/install_deps.sh

```diff
@@ -19,36 +19,23 @@
 function install_linux_deps() {
   if [[ -x "$(command -v apt)" ]]; then
       # for Ubuntu 18.04
-      sudo apt install -y g++ gcc make lcov libtool m4 autoconf automake ccache libssl-dev zlib1g-dev libboost-regex-dev \
-          libboost-program-options-dev libboost-system-dev libboost-filesystem-dev \
-          libboost-serialization-dev python3-dev libboost-python-dev libcurl4-openssl-dev gfortran libtbb-dev libzstd-dev libaio-dev \
+      sudo apt install -y g++ gcc make lcov libtool m4 autoconf automake ccache libssl-dev zlib1g-dev \
+          python3-dev libcurl4-openssl-dev gfortran libaio-dev \
           uuid-dev libpulse-dev
+      sudo pip3 install conan
```

Comment:
Can you renew the deps with build/docker/*/Dockerfile

Reply:
done